### DISCUSSION TITLE
Read only mode?

### DISCUSSION THREAD

**Discussion by TheNomad11** [docmost/docmost#316] [General]
When I use Docmost to save articles I would prefer to protect them from accidently editing. So I wonder if read-only mode would be an option for future releases or if it is available already? Thanks! Looks very promising so far!

**Comment by thalion0** [comment]
Hi @TheNomad11,

same thoughts as mine. I expect many mistakenly changes without readonly mode.

**Comment by almereyda** [comment]
As a workaround, you could always remove edit permissions from yourself.

**Comment by leadvic** [comment]
I agree, this is super important, it would be great if we could have a padlock that we have to press before being able to start editing the page, as that way you would avoid missediting the page.

I think this would be great for any human being that can missclick, or has a baby that likes pressing buttons or have cats that randomly jump over the keyboard 😅

**Comment by voltronunited** [comment]
and also because most users are used to seeing a read-only view first (it's hard to change habits).

**Comment by kasparlov** [comment]
Yes, please add something like an "edit" button. Looking up docs, I always start editing by mistake, especially on mobile.

**Comment by fuscodev** [comment]
I did it in PR #666

**Comment by BeanHubbleday** [comment]
This would be an excellent addition, looks like Phil is getting behind it now which is brilliant!
